docs/reference/emails.md

```diff
@@ -0,0 +1,418 @@
+# After the colon, there's a link to https://accounts.firefox.com/support
+subscriptionSupportContact-plaintext = Thank you for subscribing to { $productName }. If you have any questions about your subscription or need more information about { $productName }, please contact us:
+```
+
+If the element you need translated is already wrapped in a non-MJML tag, like `b`, or `li`, supply the `data-l10n-id` on that element instead of creating an extra `span` DOM element.
+
+```
+<b data-l10n-id="payment-details">Payment details:</b>
+```
+
+You do _not_ need a `data-l10n-id` on strings that only contains a variable since the variable won't be localized.
+```
+<mj-text css-class="code-large"><%- code %></mj-text>
+```
+
+Fluent will overlay the translation onto the source fragment preserving attributes like `class` and `href` from the source and adding translations for the elements inside.
+
+:::warning
+If you need to change a string, you must also update the Fluent ID. Generally speaking, we just append a `-v2` or `-v3` to the string if it's a rewording or we create a new ID entirely if the copy is significantly different. We must do this because IDs are saved in Pontoon and tied to translations for the original string.
+
+If you change a variable name and not the string text around it, technically you also need a new ID since the string is not identical. However, to not lose existing translations, you can also find-and-replace the variable name in that ID across locales in the l10n repo directly before or after your PR in `fxa` is merged. You must coordinate with the l10n team if you plan to do this.
+:::
+
+At the time of writing, Storybook is our way to preview or manually check the English strings we ultimately pass to translators, and our tests cover the English fallback copy.
+### Images and localizing alt text
+
+Since we must pass Fluent an FTL ID for each string to be localized, localizing alt text for images is tricky. We use [`mj-html-attributes`](https://documentation.mjml.io/#mj-html-attributes) to add a custom attribute where we need them, and all of our images are given these HTML attributes in an `images.mjml` file pulled into every layout file:
+
+```html
+<mj-selector path=".mozilla-logo a">
+  <mj-html-attribute name="data-l10n-id">subplat-footer-mozilla-logo</mj-html-attribute>
+</mj-selector>
+```
+
+In MJML layout:
+
+```html
+<mj-image css-class="mozilla-logo"
+  width="120px"
+  href="https://www.mozilla.org"
+  src="https://accounts-static.cdn.mozilla.net/product-icons/mozilla-logo.png"
+  alt="Mozilla logo"
+  title="Mozilla">
+</mj-image>
+```
+
+In corresponding FTL files:
+
+```
+subplat-footer-mozilla-logo = <img data-l10n-name="mozilla-logo" alt="{ -brand-mozilla } logo">
+```
+
+We could have technically localized strings this way as well rather than wrap text elements in `span`s, but that would have been significantly messier and confusing.
+
```

Comment:
We should add in a reminder that if no width is provided in the mj-image tag that the image will use the parent column width, as we saw in this PR - https://github.com/mozilla/fxa/pull/11840  😅

Reply:
Thanks for calling this out, I definitely forgot this piece. I added a note to the top of this section about it with a link to your PR, as well as added a comment within the MJML image example.